baselines/asset/src/v1/asset_service_client.ts.baseline

```diff
@@ -73,40 +73,34 @@ export class AssetServiceClient {
    *     your project ID will be detected automatically.
    * @param {string} [options.apiEndpoint] - The domain name of the
    *     API remote host.
+   * @param {gax.ClientConfig} [options.clientConfig] - client configuration override.
+   *     Client configuration set in the JSON config
+   *     (`src/v1/asset_service_client_config.json`)
+   *     can be overridden by passing `clientConfig` to the client constructor.
+   *     The detailed structure of the clientConfig can be found
+   *     {@link https://github.com/googleapis/gax-nodejs/blob/master/src/gax.ts#L564-L611 here}.
```

Comment:
I see a few problems here.  I don't think the reference here will stay up to date as the code in `gax.ts` changes and moves.  So pinning to `master` is unlikely to stay correct.  Second - should this be a reference to another interface?  I'm trying to look a lot closer at the reference API documentation we generate.

Reply:
Yeah it can move. I need to see if I can make a proper jsdoc link to a gax interface.

Reply:
I removed all links to gax in this PR and will take care of this separately (as discussed, will add links to a separate document).